frontend/public/config.yml

```
@@ -265,6 +266,21 @@ collections:
                   { label: Target, name: link_target, widget: string },
                 ],
             }
+      - label: 'Thank you'
+        name: 'thank_you'
+        file: 'frontend/src/content/pages/thank_you.md'
+        fields:
+          - {
+              label: 'Thank you page title',
```

yamllint

[error] wrong indentation: expected 12 but found 14

(indentation)